dist-git-to-src-git/updates.md

```diff
@@ -0,0 +1,126 @@
+# Updates of source-git repos
+
+
+## Problem
+
+- We need to convert new dist-git commits to the existing source-git repository.
+- The changes need to be reversible.
+  - => We need to get back to the dist-git structure from the source-git.
+- Possible changes:
+  - changes in the packaging files (not in spec-file, not in patch files; e.g. config-files)
+  - general changes in the spec-file (e.g. new release, new dependency)
+  - change in patches:
+    - patch added
+    - patch edited
+    - patch removed
+- Patches can be applied conditionally.
+- Users can help with the sync. (Tweak the change.)
+- No force-push possible.
+
+
+### What can differ when going back to the dist-git structure?
+
+- Be able to create the same `RPM` file?
+- Can spec-file differ? e.g. different patch names, comments
```

Comment:
they shouldn't differ, otherwise it would be pain to work with such a system for maintainers